pkgs/stdenv/cross/default.nix

```diff
@@ -48,14 +51,17 @@ in lib.init bootStages ++ [
 
       cc = if crossSystem.useiOSPrebuilt or false
              then buildPackages.darwin.iosSdkPkgs.clang
-           else if crossSystem.useAndroidPrebuilt
+           else if crossSystem.useAndroidPrebuilt or false
              then buildPackages.androidenv."androidndkPkgs_${crossSystem.ndkVer}".gcc
            else buildPackages.gcc;
 
       extraNativeBuildInputs = old.extraNativeBuildInputs
         ++ lib.optionals
              (hostPlatform.isLinux && !buildPlatform.isLinux)
              [ buildPackages.patchelf buildPackages.paxctl ]
+        ++ lib.optionals
+             (hostPlatform.isDarwin && buildPlatform.isDarwin)
+             [ buildPackages.clang ]
```

Comment:
what's this?

Reply:
We are using gcc for all cross stuff. But, gcc alone can't build binaries for some reason. It needs to call clang at some point.

Reply:
This is the error I get without it:

```
/nix/store/...-gcc-wrapper-7.3.0/bin/as: assembler (clang) not installed
```